orchestrator/careplancontributor/service.go

```diff
@@ -128,6 +128,12 @@ func (s Service) RegisterHandlers(mux *http.ServeMux) {
 		log.Info().Msgf("Redirecting to %s", s.frontendUrl)
 		http.Redirect(response, request, s.frontendUrl, http.StatusFound)
 	})
+
+	// Logout endpoint
+	mux.HandleFunc(basePath+"/zorgplatform/logout", func(response http.ResponseWriter, request *http.Request) {
```

Comment:
I'd have this as authenticated endpoint (`s.withSession`), then you know for sure there's actually a session to destroy